app/services/customers/create_service.rb

```diff
@@ -121,7 +124,7 @@ def create(**args)
       end
 
       # NOTE: handle configuration for configured payment providers
-      billing_configuration = args[:provider_customer]&.to_h&.merge(payment_provider: args[:payment_provider])
+      billing_configuration = args[:provider_customer]&.to_h&.merge(payment_provider: args[:payment_provider], payment_provider_code: args[:payment_provider_code])
```

Comment:
Layout/LineLength: Line is too long. [163/120] (https://rubystyle.guide#max-line-length)